### ISSUE TITLE
[Bug] Crashing when pondering a fluid pipe.

### ISSUE THREAD

**Issue by taluDais** [iglee42/CreateCasing#83]
The game crashes in the middle of pondering of a fluid pipe when the mechanical pipe is about to appear.

 - OS: Windows 11 build 22621,3737
 - Java version: Eclipse Adoptium JDK-17.0,11.9
 - Minecraft version: 1.20.1
 - Forge version: 47.3.5
 - Create version: 0.5.1.f
 - Create: Encased version: 1.6.1
 - No other mods were loaded
 
And here's a crash log: 
https://mclo.gs/AwA3xRG

**Comment by iglee42** [comment]
I think it's crash from create because Create Encased don't modify fluid pipe code

**Comment by taluDais** [comment]
i did test it with vanilla create and there was no issue

**Comment by taluDais** [comment]
and also cogwheels are appearing with a mechanical pump

**Comment by iglee42** [comment]
I don't understand where the crash could come from

**Comment by omgimanerd** [comment]
+1 to this issue. I replicated it with 

Minecraft version: 1.20.1
Forge loader: 47.3.0
Create: 0.5.1f
Create Encased: 1.20.1-1.6.1

The culprit line here:
```
com.google.common.util.concurrent.UncheckedExecutionException: java.lang.IllegalArgumentException: Cannot get property BooleanProperty{name=down, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{create:mechanical_pump}
```
seems to indicate that a property of the mechanical pump is causing the crash. If I were to guess, it's because mechanical pumps call some of the cogwheel code (since they have a cogwheel)?

**Comment by iglee42** [comment]
Fixed in the next update

**Comment by omgimanerd** [comment]
Will this bugfix be released on CF for version 1.20.1? Or only future versions?

**Comment by iglee42** [comment]
It will be released for 1.18.2/1.19.2/1.20.1

**Comment by iglee42** [comment]
Update pushed on curseforge : https://create-encased.iglee.fr
